src/components/NavBar/index.tsx

```diff
@@ -1,70 +1,64 @@
 import { useState } from 'react';
 import { Link } from 'react-router-dom';
-import { faCaretDown, faTools } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { pages } from '../../assets/pages';
 
-import './NavBar.css';
-import NavBarTools from './NavBarTools';
+import NavBarStyle from './NavBar.module.css';
 import NavBarLogo from './NavBarLogo';
+import DropDownMenu from './DropDownMenu';
 
-interface PlatformType {
-  id: string;
-  header: string;
-  style: string;
-}
 interface NavBarPropsType {
   toggleMainWidth: () => void;
-  mini: Boolean;
-  platform: PlatformType;
+  mini: boolean;
+  platform: {
```

Comment:
you use it in couple of places - create the type in the platform file and import is